server/runtime/usr/local/bin/start-server

```diff
@@ -4,14 +4,6 @@ set -e
 # Save the environment to be able to restore it in the FCGI daemon (used in /usr/local/bin/mapserv_wrapper)
 env | sed -e 's/.\+/export "\0"/' > /tmp/init_env
 
-if [[ "${UID}" != 0 ]]
```

Comment:
That would break backward compatibility. We have a lot of compositions/charts/... that would break if you don't listen on 8080. We should keep this behavior by default and add an environment variable to disable it.

Reply:
Ok, `LISTEN_PORT_80` environment variable has been introduced. It has to be set to `1` for the container to listen on default port 80.